tests/conftest.py

```diff
@@ -0,0 +1,103 @@
+from typing import AsyncGenerator, Callable, Optional
+from uuid import UUID, uuid4
+
+import httpx
+import pytest
+
+from botx import (
+    BotAccount,
+    BotAccountWithSecret,
+    Chat,
+    ChatTypes,
+    IncomingMessage,
+    UserDevice,
+    UserSender,
+)
+
+
+@pytest.fixture
+def bot_id() -> UUID:
+    return UUID("24348246-6791-4ac0-9d86-b948cd6a0e46")
+
+
+@pytest.fixture
+def user_huid() -> UUID:
+    return UUID("6d4b5b6e-ae32-4988-be5d-47baa7100c66")
+
+
+@pytest.fixture
+def chat_id() -> UUID:
+    return UUID("211c57d5-3439-4952-aa1d-729f70dd5871")
+
+
+@pytest.fixture
+def host() -> str:
+    return "cts.example.com"
+
+
+@pytest.fixture
+def bot_account(host: str, bot_id: UUID) -> BotAccountWithSecret:
+    return BotAccountWithSecret(
+        id=bot_id,
+        host=host,
+        secret_key="bee001",
+    )
+
+
+@pytest.fixture
+async def httpx_client() -> AsyncGenerator[httpx.AsyncClient, None]:
```

Comment:
Unnecessary fixture, bot can create default httpx client by itself.

Reply:
Thanks. I delete this fixture.